package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Practice_Assignments",
+  "version": "1.0.0",
+  "description": "This is practice assignment using campaign use case",
+  "main": "server.js",
+  "scripts": {
+    "start": "babel-node --presets es2015 server.js"
```

Comment:
start script should be either of these : `node server.js`, `nodemon server.js` or more appropriately `forever start server.js`